spec/migrations/20150625220141_fix_serialized_reports_for_rails_four_spec.rb

```diff
@@ -1,9 +1,25 @@
 require_migration
 
 describe FixSerializedReportsForRailsFour do
+  before do
+    # Create the various classes that are represented in the serialized data
+    class ::MiqReport; end
+    class ::Ruport
+      class Data
+        class Table; end
+        class Record; end
+      end
+    end
+  end
+
+  after do
+    Object.send(:remove_const, "MiqReport")
+    Object.send(:remove_const, "Ruport")
+  end
```

Comment:
I think you can use `stub_const` to return `Class.new` with those names instead of actually defining a top level class and cleaning up afterward. I remember doing this somewhere.